_config.template.yml

```diff
@@ -82,10 +82,13 @@ fonts: Roboto, "Helvetica Neue", Helvetica, "PingFang SC", "Hiragino Sans GB", "
 # You can find demos on http://github.hubspot.com/pace/docs/welcome
 # Warning: Line color mustn't delete or not define!
 #     style available values:
-#       barber-shop, big-counter, bounce, center-circle, center-radar, center-simple,
+#       barber-shop, big-counter, bounce, center-atom, center-circle, center-radar, center-simple,
 # 	    corner-indicator, fill-left, fill-left-nocolor, flat-top, loading-bar, macosx
+# Or you can use your own style,just set the style value to "mycss",
+# and put your own css style to source/css/pace/my.css
```

Comment:
The `barber-shop` style being implemented in the `barber-shop.css` file, the `big-counter` style being implemented in `big-counter.css` file, and so on... the `mycss` style should be implemented in `mycss.css` file (not `my.css`), or in the other way, `my.css` file should implement the `my` style (not `mycss`).

That said, `my` is not a appropriate word. I would have called the style `custom` and its corresponding file `custom.css`. What do you think?